ConfigManager.cs

```diff
@@ -24,8 +25,8 @@ public class Config
 		}
 		catch
 		{
-			Program.LogAsync("Config", "Failed to parse config file.", LogSeverity.Critical);
-			return null;
+            Log.Fatal("Failed to parse config file.");
```

Comment:
Use with the change above
Provide more information about the issue, so the user can have a general idea of what the problem is

```suggestion
            Log.Fatal($"Failed to parse config file: {e.Message}");
```